script/PR_internal_chart_template.md

```diff
@@ -0,0 +1,19 @@
+### Description of the change
+
+This is an automatic PR for synchronizing the changes performed externally at the [bitnami/kubeapps](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps) official chart.
+
+### Benefits
+
+The Kubeapps [development chart](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps) will get the newest changes, including image version updates and other updates.
+
+### Possible drawbacks
+
+Even unlikely, some changes implemented by the Bitnami team may not be 100% compatible with the current Kubeapps version.
```

Comment:
This part is OK since we need to update to cope, I guess.
```suggestion
Although unlikely, some changes implemented by the Bitnami team may not be 100% compatible with the current Kubeapps version.
```